### DISCUSSION TITLE
It this the current way to pass data to a relation manager action?

### DISCUSSION THREAD

**Discussion by ghost** [filamentphp/filament#11536] [Help] [answered]
### Package

Actions

### Package Version

3

### How can we help you?

In my relation manager I want to pass the `id` to the form:

```php
            ->headerActions([
                Tables\Actions\CreateAction::make()
                    ->fillForm([
                        'business_id' => $this->ownerRecord->id,
                    ]),
            ]);
```

This works, I'm just wondering if this the correct approach?

**Comment by leandrocfe** [comment, marked as answer]
Yes https://filamentphp.com/docs/3.x/panels/resources/relation-managers#accessing-the-relationships-owner-record
You can also do this `$this->getOwnerRecord()->getKey()`

**Comment by tigusigalpa** [comment]
Yes it's possible: https://stackoverflow.com/a/79640484/724816
